Automattic/jetpack
# Show classic admin interface profile settings notice

Fixes https://github.com/Automattic/dotcom-forge/issues/7877

## Proposed changes:

This PR adds a banner that helps explain that updates made on `/wp-admin/profile.php` only sync to sites with the Classic admin-interface style. Classic admin-interface sites are "untangled" from `/me` and `/me/account` and do not sync.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Apply this patch to your sandbox
2. Test this on a site that uses the "Classic" admin interface
3. Go to `/wp-admin/profile.php`, check that you see this banner:

![image](https://github.com/Automattic/jetpack/assets/8511199/367c61e7-474b-4ea4-894e-dc757a3b40cd)

4. Check that the link goes to https://wordpress.com/support/dashboard/#set-the-admin-interface-style
